_pages/oidc/getting-started.md

```diff
@@ -0,0 +1,86 @@
+---
+title: OpenID Connect
+lead: >
+   <a class="usa-link usa-link usa-link--external" href="http://openid.net">OpenID Connect</a> is a simple identity layer built on top of the OAuth 2.0 protocol. Login.gov supports <a class="usa-link usa-link usa-link--external" href="http://openid.net/specs/openid-connect-core-1_0.html">version 1.0</a> of the specification and conforms to the <a class="usa-link usa-link usa-link--external" href="https://openid.net/wg/igov">iGov Profile</a>.
+redirect_from:
+  - /openid-connect/
+sidenav:
+  - text: Getting started
+    href: "oidc/getting-started/"
+    links:
+      - text: Choosing an authentication method
+        href: "#choosing-an-authentication-method"
+      - text: Set up a Sandbox account
+        href: "#set-up-a-sandbox-account"
+      - text: Auto-discovery
+        href: "#auto-discovery"
+      - text: Example application
+        href: "#example-application"
+  - text: Authorization
+    href: "oidc/authorization/"
+  - text: Token
+    href: "oidc/#token"
+  - text: User info
+    href: "oidc/#user-info"
+  - text: Certificates
+    href: "oidc/#certificates"
+  - text: Logout
+    href: "oidc/#logout"
+  - text: Example application
+    href: "oidc/#example-application"
```

Comment:
is there any way to reuse this sidenav between pages? having to change 2+ other pages if a heading gets renamed seems like a lot of work

Reply:
hmm not sure... each page nav will be slightly different because the page you are on will have the sub sections on the current page 'expanded' and sub sections on other pages hidden